airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -74,22 +74,26 @@ class SecretsManagerBackend(BaseSecretsBackend, LoggingMixin):
     However, these lists can be extended using the configuration parameter ``extra_conn_words``.
 
     :param connections_prefix: Specifies the prefix of the secret to read to get Connections.
-        If set to None (null), requests for connections will not be sent to AWS Secrets Manager
+        If set to None (null value in the configuration), requests for connections will not be
+        sent to AWS Secrets Manager. If you don't want a connections_prefix, set it as an empty string
     :type connections_prefix: str
     :param variables_prefix: Specifies the prefix of the secret to read to get Variables.
-        If set to None (null), requests for variables will not be sent to AWS Secrets Manager
+        If set to None (null value in the configuration), requests for variables will not be sent to
+        AWS Secrets Manager. If you don't want a variables_prefix, set it as an empty string
     :type variables_prefix: str
     :param config_prefix: Specifies the prefix of the secret to read to get Configurations.
-        If set to None (null), requests for configurations will not be sent to AWS Secrets Manager
+        If set to None (null value in the configuration), requests for configurations will not be sent to
+        AWS Secrets Manager. If you don't want a config_prefix, set it as an empty string
```

Comment:
We don't need emptry string -- we can set it s `null` in the JSON config

Reply:
If it's null, then it shouldn't get connections. And this must be maintained because is what the previous behavior was:
https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/secrets-backends/aws-secrets-manager.html#optional-lookup
https://github.com/apache/airflow/blob/eda538f56cb2dc2728d303acddb42841fe419c36/airflow/providers/amazon/aws/secrets/secrets_manager.py#L57
https://github.com/apache/airflow/blob/eda538f56cb2dc2728d303acddb42841fe419c36/airflow/providers/amazon/aws/secrets/secrets_manager.py#L112

Reply:
Looks like it was added in https://github.com/apache/airflow/pull/12143

Reply:
Since we had the default of following:

https://github.com/apache/airflow/blob/acd171df988d8078cbcb0a80848114e5fc4e0e6d/airflow/providers/amazon/aws/secrets/secrets_manager.py#L106-L108

the only way to explicitly say NO I don't want to use Secrets Backend for Connections is to set it to `null`.

(I haven't seen exact details of this PR yet but just chiming in to give reasoning of previous logic)

cc @fhoda